generators/app/index.js

```diff
@@ -383,7 +506,7 @@ module.exports = class extends Generator {
                     name: 'checkJavaScript',
                     message: 'Enable JavaScript type checking in \'jsconfig.json\'?',
                     default: false
-                }).then(function (strictJavaScriptAnswer) {
+                }).then(strictJavaScriptAnswer => {
```

Comment:
they are too many changes in this PR. makes it hard to review. please make sure the changes are only for dashboard. If they are other changes please send separate PRs. in general sending smaller PRs makes it easier to review them